pkg/storage/unified/resource/bulk.go

```diff
@@ -250,6 +250,18 @@ func (s *server) BulkProcess(stream resourcepb.BulkStore_BulkProcessServer) erro
 		rsp.Error = AsErrorResult(runner.err)
 	}
 
+	// Trigger immediate index rebuild for successful bulk imports
+	// This ensures search indexes are updated immediately after the data is committed
+	if rsp.Error == nil && s.search != nil {
```

Comment:
This is what we used to do before introducing "last import time" approach. It doesn't work in our cloud setup where we deploy storage and search pods separately.

Removing call to "updateLastImportTime" in bulk.go breaks our setup and will leave indexes stale after bulk import.